### ISSUE TITLE
Unhandled exception when typing "c " in interactive mode

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbemtools#192]
In section 2.2 of the RTD docs, there is an example command in interactive mode:

`> c --help`

This command displays an exception and brings the pywbemcli into a state where I did not find any way to exit the interactive mode (on MacOS). Any action displays the exception again. I had to terminate the terminal window to get out of it.

Here is the exception displayed, which happens after typing `c ` (i.e. at the first blank):

```
$ pywbemcli
Enter 'help' for help, <CTRL-D> or ':q' to exit pywbemcli.

Unhandled exception in event loop:
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/eventloop/posix.py", line 154, in _run_task
    t()
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/eventloop/context.py", line 115, in new_func
    return func(*a, **kw)
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/application/application.py", line 376, in redraw
    self._redraw()
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/application/application.py", line 444, in _redraw
    self.renderer.render(self, self.layout)
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/renderer.py", line 578, in render
    screen.draw_all_floats()
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/screen.py", line 232, in draw_all_floats
    functions[0][1]()
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 700, in _draw_float
    width = fl.content.preferred_width(write_position.width).preferred
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 2173, in preferred_width
    return self.content.preferred_width(max_available_width)
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 1339, in preferred_width
    dont_extend=self.dont_extend_width())
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 1380, in _merge_dimensions
    preferred = get_preferred()
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 1328, in preferred_content_width
    max_available_width - total_margin_width)
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 46, in preferred_width
    menu_meta_width = self._get_menu_meta_width(500, complete_state)
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 110, in _get_menu_meta_width
    if self._show_meta(complete_state):
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 94, in _show_meta
    return any(c.display_meta_text for c in complete_state.completions)
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 94, in <genexpr>
    return any(c.display_meta_text for c in complete_state.completions)
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/completion/base.py", line 92, in display_meta_text
    return fragment_list_to_text(self.display_meta)
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/completion/base.py", line 86, in display_meta
    return to_formatted_text(self._display_meta)
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/formatted_text/base.py", line 50, in to_formatted_text
    'HTML, ANSI or a FormattedText instance. Got %r' % value)

Exception No formatted text. Expecting a unicode object, HTML, ANSI or a FormattedText instance. Got 'Hostname or IP address with scheme of the WBEMServer in format:\n[{scheme}://]{host}[:{port}]\nThe server parameter is conditionally optional (see --name)\n* Scheme: must be "https" or "http" [Default: "https"]\n* Host: defines short/fully qualified DNS hostname, literal IPV4 address (dotted), or literal IPV6 address\n* Port: (optional) defines WBEM server port to be used [Defaults: 5988(HTTP) and 5989(HTTPS)].\n(EnvVar: PYWBEMCLI_SERVER).'
Press ENTER to continue...     
```

**Comment by KSchopmeyer** [comment]
I found and fixed the 'c' but now let me see if I can recreate the failure on linux/windows

**Comment by KSchopmeyer** [comment]
Andy could you retest with python 3.

It fails for me on multiple operating systems with python 2.7.  I need to double check to be sure that it happens on Linux but I know it happens on Windows and python 2.7.

It does not happen to me on python 3.

Note that this is not tied to that particular command.  I have had it happen with completely correct commands and also part way through entering the parameter sequence.

**Comment by KSchopmeyer** [comment]
See issue #223 which is the same issue with multiple examples.

**Comment by andy-maier** [comment]
On MacOS with Python 2.7, with the latest master version of pywbemtools, I still get the exception, when I type `c ` (`c`followed by blank) in the interactive mode.

Python 2.7.16
click-repl                         0.1.6        
click-spinner                      0.1.8        
prompt-toolkit                     2.0.9 

```
(pywbem27) dyn-9-152-221-78:pywbemtools maiera$ pywbemcli
Enter 'help' for help, <CTRL-D> or ':q' to exit pywbemcli.

Unhandled exception in event loop:
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/eventloop/posix.py", line 154, in _run_task
    t()
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/eventloop/context.py", line 115, in new_func
    return func(*a, **kw)
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/application/application.py", line 376, in redraw
    self._redraw()
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/application/application.py", line 444, in _redraw
    self.renderer.render(self, self.layout)
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/renderer.py", line 578, in render
    screen.draw_all_floats()
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/screen.py", line 232, in draw_all_floats
    functions[0][1]()
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 700, in _draw_float
    width = fl.content.preferred_width(write_position.width).preferred
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 2173, in preferred_width
    return self.content.preferred_width(max_available_width)
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 1339, in preferred_width
    dont_extend=self.dont_extend_width())
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 1380, in _merge_dimensions
    preferred = get_preferred()
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 1328, in preferred_content_width
    max_available_width - total_margin_width)
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 46, in preferred_width
    menu_meta_width = self._get_menu_meta_width(500, complete_state)
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 110, in _get_menu_meta_width
    if self._show_meta(complete_state):
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 94, in _show_meta
    return any(c.display_meta_text for c in complete_state.completions)
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 94, in <genexpr>
    return any(c.display_meta_text for c in complete_state.completions)
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/completion/base.py", line 92, in display_meta_text
    return fragment_list_to_text(self.display_meta)
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/completion/base.py", line 86, in display_meta
    return to_formatted_text(self._display_meta)
  File "/Users/maiera/virtualenvs/pywbem27/lib/python2.7/site-packages/prompt_toolkit/formatted_text/base.py", line 50, in to_formatted_text
    'HTML, ANSI or a FormattedText instance. Got %r' % value)

Exception No formatted text. Expecting a unicode object, HTML, ANSI or a FormattedText instance. Got 'Hostname or IP address with scheme of the WBEMServer in format:\n[{scheme}://]{host}[:{port}]\nThe server parameter is conditionally optional (see --name)\n* Scheme: must be "https" or "http" [Default: "https"]\n* Host: defines short/fully qualified DNS hostname, literal IPV4 address (dotted), or literal IPV6 address\n* Port: (optional) defines WBEM server port to be used [Defaults: 5988(HTTP) and 5989(HTTPS)].\n(EnvVar: PYWBEMCLI_SERVER).'
Press ENTER to continue...
```
```

**Comment by KSchopmeyer** [comment]
Here are a number of additional exceptions from issue # 224.  We will close that one as duplicate

In the REPL mode we get repeasted unhandled exceptions in the  prompt_toolkit event loop.

Note that this is same as issue #192

The driving exception is 

  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/formatted_text/base.py", line 50, in to_formatted_text
    'HTML, ANSI or a FormattedText instance. Got %r' % value)

Exception No formatted text. Expecting a unicode object, HTML, ANSI or a FormattedText instance. Got 'Show only local properties of the instances. This subcommand may use either pull or traditional operations depending on the server and the "--use--pull-ops" general option. If pull operations are used, this parameters will not be included, even if specified. If traditional operations are used, some servers do not process the parameter.'

This does not depend on any particular subcommand sequence and imanifests only after a at least one successful command has been executed.

===========================
Examples
pywbemcli> instance count                                                       
Count of instances per class
+---------+---------+
| Class   |   count |
|---------+---------|
| CIM_Foo |       3 |
+---------+---------+

Unhandled exception in event loop:
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/eventloop/posix.py", line 154, in _run_task
    t()
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/eventloop/context.py", line 115, in new_func
    return func(*a, **kw)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/application/application.py", line 376, in redraw
    self._redraw()
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/application/application.py", line 444, in _redraw
    self.renderer.render(self, self.layout)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/renderer.py", line 578, in render
    screen.draw_all_floats()
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/screen.py", line 232, in draw_all_floats
    functions[0][1]()
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 700, in _draw_float
    width = fl.content.preferred_width(write_position.width).preferred
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 2173, in preferred_width
    return self.content.preferred_width(max_available_width)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 1339, in preferred_width
    dont_extend=self.dont_extend_width())
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 1380, in _merge_dimensions
    preferred = get_preferred()
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 1328, in preferred_content_width
    max_available_width - total_margin_width)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 46, in preferred_width
    menu_meta_width = self._get_menu_meta_width(500, complete_state)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 110, in _get_menu_meta_width
    if self._show_meta(complete_state):
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 94, in _show_meta
    return any(c.display_meta_text for c in complete_state.completions)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 94, in <genexpr>
    return any(c.display_meta_text for c in complete_state.completions)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/completion/base.py", line 92, in display_meta_text
    return fragment_list_to_text(self.display_meta)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/completion/base.py", line 86, in display_meta
    return to_formatted_text(self._display_meta)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/formatted_text/base.py", line 50, in to_formatted_text
    'HTML, ANSI or a FormattedText instance. Got %r' % value)

Exception No formatted text. Expecting a unicode object, HTML, ANSI or a FormattedText instance. Got 'Optional property definitions of the form name=value.Multiple definitions allowed, one for each property to be included in the createdinstance. Array property values defined by comma-separated-values. EmbeddedInstance not allowed.'


+---------+---------+
| Class   |   count |
|---------+---------|
| CIM_Foo |       3 |
+---------+---------+
pywbemcli> instance enumerate                                                   
Usage: pywbemcli  instance enumerate [COMMAND-OPTIONS] CLASSNAME
Try "pywbemcli  instance enumerate -h" for help.

Error: Missing argument "CLASSNAME".

Unhandled exception in event loop:
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/eventloop/posix.py", line 154, in _run_task
    t()
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/eventloop/context.py", line 115, in new_func
    return func(*a, **kw)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/application/application.py", line 376, in redraw
    self._redraw()
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/application/application.py", line 444, in _redraw
    self.renderer.render(self, self.layout)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/renderer.py", line 578, in render
    screen.draw_all_floats()
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/screen.py", line 232, in draw_all_floats
    functions[0][1]()
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 700, in _draw_float
    width = fl.content.preferred_width(write_position.width).preferred
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 2173, in preferred_width
    return self.content.preferred_width(max_available_width)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 1339, in preferred_width
    dont_extend=self.dont_extend_width())
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 1380, in _merge_dimensions
    preferred = get_preferred()
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 1328, in preferred_content_width
    max_available_width - total_margin_width)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 46, in preferred_width
    menu_meta_width = self._get_menu_meta_width(500, complete_state)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 110, in _get_menu_meta_width
    if self._show_meta(complete_state):
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 94, in _show_meta
    return any(c.display_meta_text for c in complete_state.completions)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 94, in <genexpr>
    return any(c.display_meta_text for c in complete_state.completions)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/completion/base.py", line 92, in display_meta_text
    return fragment_list_to_text(self.display_meta)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/completion/base.py", line 86, in display_meta
    return to_formatted_text(self._display_meta)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/formatted_text/base.py", line 50, in to_formatted_text
    'HTML, ANSI or a FormattedText instance. Got %r' % value)

Exception No formatted text. Expecting a unicode object, HTML, ANSI or a FormattedText instance. Got 'Show only local properties of the instances. This subcommand may use either pull or traditional operations depending on the server and the "--use--pull-ops" general option. If pull operations are used, this parameters will not be included, even if specified. If traditional operations are used, some servers do not process the parameter.'


==================================
pywbemcli> instance enumerate CIM_Foo                                           
instance of CIM_Foo {
   InstanceID = "CIM_Foo1";
   IntegerProp = 1;
};

instance of CIM_Foo {
   InstanceID = "CIM_Foo2";
   IntegerProp = 2;
};

instance of CIM_Foo {
   InstanceID = "CIM_Foo3";
};


Unhandled exception in event loop:
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/eventloop/posix.py", line 154, in _run_task
    t()
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/eventloop/context.py", line 115, in new_func
    return func(*a, **kw)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/application/application.py", line 376, in redraw
    self._redraw()
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/application/application.py", line 444, in _redraw
    self.renderer.render(self, self.layout)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/renderer.py", line 578, in render
    screen.draw_all_floats()
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/screen.py", line 232, in draw_all_floats
    functions[0][1]()
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 700, in _draw_float
    width = fl.content.preferred_width(write_position.width).preferred
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 2173, in preferred_width
    return self.content.preferred_width(max_available_width)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 1339, in preferred_width
    dont_extend=self.dont_extend_width())
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 1380, in _merge_dimensions
    preferred = get_preferred()
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/containers.py", line 1328, in preferred_content_width
    max_available_width - total_margin_width)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 46, in preferred_width
    menu_meta_width = self._get_menu_meta_width(500, complete_state)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 110, in _get_menu_meta_width
    if self._show_meta(complete_state):
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 94, in _show_meta
    return any(c.display_meta_text for c in complete_state.completions)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/layout/menus.py", line 94, in <genexpr>
    return any(c.display_meta_text for c in complete_state.completions)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/completion/base.py", line 92, in display_meta_text
    return fragment_list_to_text(self.display_meta)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/completion/base.py", line 86, in display_meta
    return to_formatted_text(self._display_meta)
  File "/home/kschopmeyer/virtualenvs/pywbemtoolstmp/local/lib/python2.7/site-packages/prompt_toolkit/formatted_text/base.py", line 50, in to_formatted_text
    'HTML, ANSI or a FormattedText instance. Got %r' % value)

Exception No formatted text. Expecting a unicode object, HTML, ANSI or a FormattedText instance. Got 'Show only local properties of the instances. This subcommand may use either pull or traditional operations depending on the server and the "--use--pull-ops" general option. If pull operations are used, this parameters will not be included, even if specified. If traditional operations are used, some servers do not process the parameter.'

**Comment by KSchopmeyer** [comment]
SOLUTION: It appears that this is an issue that arrived with prompt_toolkit 2.0.x. We know it fails with 2.0.9 and have not reviewed earlier versions.  There are two possible solutions:

1. Identify the issue in prompt_toolkit 2.0.x and request fix.

2. In the mean time, restrict users of python 2.7 to prompt-toolkit less than 2.0.0.

I propose that we do both but we should be able to release version 0.5.0 by restricting the maximum version of prompt-toolkit to LT 2.0.0

**Comment by andy-maier** [comment]
FIxed in PR #234.
